src/main/java/io/appium/java_client/pagefactory/bys/builder/AppiumByBuilder.java

```diff
@@ -73,13 +78,13 @@ protected AppiumByBuilder(String platform, String automation) {
     }
 
     private static Method[] prepareAnnotationMethods(Class<? extends Annotation> annotation) {
-        List<String> targeAnnotationMethodNamesList =
+        List<String> targetAnnotationMethodNamesList =
             getMethodNames(annotation.getDeclaredMethods());
-        targeAnnotationMethodNamesList.removeAll(METHODS_TO_BE_EXCLUDED_WHEN_ANNOTATION_IS_READ);
-        Method[] result = new Method[targeAnnotationMethodNamesList.size()];
-        for (String methodName : targeAnnotationMethodNamesList) {
+        targetAnnotationMethodNamesList.removeAll(METHODS_TO_BE_EXCLUDED_WHEN_ANNOTATION_IS_READ);
+        Method[] result = new Method[targetAnnotationMethodNamesList.size()];
```

Comment:
could this be done in more functional style (e.g. using streams)?

Reply:
Sure, done.